node_cli/realm/realmAuth.js

```diff
@@ -0,0 +1,43 @@
+const Realm = require("realm");
+
+const REALM_APP_ID = "realmtutorials-dcyrc";
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
+
+let user;
+
+async function login(email, password) {
+  const credentials = Realm.Credentials.emailPassword(email, password);
+  try {
+    return await app.logIn(credentials).then(async (u) => {
+      user = u;
+      return user;
+    });
+  } catch (e) {
+    console.log(e);
+
+    //TODO: how to handle bad password vs. register new user
```

Comment:
Can we separate them into separate methods? e.g. `login(email, password)` and `register(email, password)`

That would make the logic split clear and users would be more confident knowing that they register a new account vs needing to know existing credentials.